conu/utils/probes.py

```diff
@@ -104,10 +110,13 @@ def _run(self):
                     else:
                         raise result
                 elif not (result == self.expected_retval):
+                    logger.info("waiting for process to end")
```

Comment:
Something like `waiting for process to end ...`

Reply:
can do